src/lib/utils/helpers.ts

```diff
@@ -30,25 +36,13 @@ export const getPaddingValues = (padding?: string | number, paddingLeft?: string
         left: 0
     }
 
-    if (!padding && paddingLeft && paddingRight) {
+    if (!padding) {
         return {
             ...zeroPadding,
-            right: extractNumberFromStyle(paddingRight.toString()),
-            left: extractNumberFromStyle(paddingLeft.toString())
-        }
-    }
-
-    if (!padding && paddingLeft) {
-        return {
-            ...zeroPadding,
-            left: extractNumberFromStyle(paddingLeft.toString())
-        }
-    }
-
-    if (!padding && paddingRight) {
-        return {
-            ...zeroPadding,
-            right: extractNumberFromStyle(paddingRight.toString())
+            right: paddingRight ? extractNumberFromStyle(paddingRight.toString()) : undefined,
```

Comment:
move check logic to extractNumberFromStyle,
then you ll have just `extractNumberFromStyle(paddingTop)`